tests/cli_test.ts

```diff
@@ -290,3 +291,66 @@ Deno.test({
   sanitizeOps: false,
   sanitizeResources: false,
 });
+
+Deno.test({
+  name: "fresh-init error(help)",
+  async fn(t) {
+    const includeText = "fresh-init";
+
+    await t.step(
+      "execute invalid init command (deno run -A init.ts)",
+      async () => {
+        const cliProcess = Deno.run({
+          cmd: ["deno", "run", "-A", "init.ts"],
+          stderr: "piped",
+        });
+        const { code } = await cliProcess.status();
+        cliProcess.close();
+        assertEquals(code, 1);
+
+        const rawError = await cliProcess.stderrOutput();
+        const errorString = new TextDecoder().decode(rawError);
+
+        assertStringIncludes(errorString, includeText);
+      },
+    );
+
+    await t.step(
+      "execute invalid init command (deno run -A init.ts -f)",
+      async () => {
+        const cliProcess = Deno.run({
+          cmd: ["deno", "run", "-A", "init.ts", "-f"],
+          stderr: "piped",
+        });
+        const { code } = await cliProcess.status();
+        cliProcess.close();
+        assertEquals(code, 1);
+
+        const rawError = await cliProcess.stderrOutput();
+        const errorString = new TextDecoder().decode(rawError);
+
+        assertStringIncludes(errorString, includeText);
+      },
+    );
+
+    await t.step(
+      "execute invalid init command (deno run -A init.ts --foo)",
+      async () => {
+        const cliProcess = Deno.run({
+          cmd: ["deno", "run", "-A", "init.ts", "--foo"],
+          stderr: "piped",
+        });
+        const { code } = await cliProcess.status();
+        cliProcess.close();
+        assertEquals(code, 1);
+
+        const rawError = await cliProcess.stderrOutput();
+        const errorString = new TextDecoder().decode(rawError);
+
+        assertStringIncludes(errorString, includeText);
+      },
+    );
+  },
+  sanitizeOps: false,
+  sanitizeResources: false,
```

Comment:
is this needed for this test?

Reply:
@lucacasonato Thanks for the code review!
Oh sorry,  this option was not necessary for this test.

I see that you have fixed this code on your end.
https://github.com/lucacasonato/fresh/pull/276/commits/386a8e3589175112f517330bc61e119c82e657cd

Thank you very much.